src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import duke.commands.Command;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Implements a version of Duke called Wyre
```

Comment:
Normally we use this grammatical mood (e.g. starting with "Implements") to describe our Java methods, but for classes you may want your header comment to focus on what it is, rather than what it does. For this example perhaps we could call it "Implementation of a Duke called Wyre." and then apply it to all the other classes. For example, you could refer to the JavaDoc headers for `MainController` and `DialogBox`.